examples/web-wagmi/src/publications/components/PublicationCard.tsx

```diff
@@ -51,13 +52,13 @@ export function CollectablePublicationCard({
         {publication.hidden ? 'This publication has been hidden' : publication.metadata.content}
       </p>
       {collectButton}
-      {'collectLimit' in publication.collectPolicy && (
+      {publication.collectPolicy.state === CollectState.COLLECT_LIMIT_REACHED && (
```

Comment:
nice, read-only state machine. 
Quite helpful given we can build discriminate `switch` on the UI and cause some `tsc` errors when we add more `CollectStates`.